src/txHandler/Base.TxHandler.ts

```diff
@@ -51,16 +72,11 @@ export default abstract class BaseTxHandler implements ITransactionHandler {
     options?: TransactionOptions | undefined
   ): Promise<any>;
   abstract handleWrapEth(amount: BigNumber): Promise<void>;
-  abstract handleApproval(
-    token: Token,
-    amount: BigNumber,
-    options?: ApprovalHandlerOptions | undefined
-  ): Promise<any>;
-  abstract handlePermit2Approval(
-    token: Token,
-    amount: BigNumber,
-    deadline: BigNumber,
-    nonce: BigNumber,
+}
+
+export abstract class BaseBulkerTxHandler extends TxHandler implements IBulkerTransactionHandler {
```

Comment:
Bulker don't have to implement `handleApproval`, just `handleBatchTransactions`

Reply:
And having `BaseBulkerTxHandler` + `BulkerTxHandler` is over engineering imo

Reply:
But the BulkerTxHandler still needs to handle token approvals before executing the batch in handleBatchTransactions() ?

Reply:
Ok removing handleApproval as the bulker only accepts permit2 approval